src/main/java/dk/kb/util/yaml/YAML.java

```diff
@@ -30,12 +30,7 @@
 import java.nio.file.AccessDeniedException;
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```

Comment:
I'm not sure how much you guys care about explicit vs wildcard imports. If it's intentional, ignore this comment. [Otherwise fix your imports, Kim would've been pleased. (: ]

Reply:
Wildcard imports bad.